templates/kurl-proxy-deployment.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if .Values.kurlProxy.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kurl-proxy-kotsadm
+  labels:
+    app: kurl-proxy-kotsadm
+    {{- include "admin-console.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: kurl-proxy-kotsadm
+  template:
+    metadata:
+      labels:
+        app: kurl-proxy-kotsadm
+        {{- include "admin-console.labels" . | nindent 8 }}
+    spec:
+      {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: proxy
+        image: {{ .Values.images.kurlProxy }}
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: NODE_PORT
+          value: "8800"
+        - name: UPSTREAM_ORIGIN
+          value: http://kotsadm:3000
+        - name: DEX_UPSTREAM_ORIGIN
+          value: http://kotsadm-dex:5556
```

Comment:
Similar question to the above. Also, dex is not even something that is shipped with this Helm chart, so i'm not sure this would ever come into play?

Reply:
Removing...